src/screens/FinancialSummary/components/PaymentCard.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
```

Comment:
Is this component used anywhere? I can't match this with anything on the [current Figma ](https://www.figma.com/file/MzpMtoYyVYFn19DpAXHEci/Mee-Panyar?node-id=2161%3A2).

If it's not used anywhere, we might wanna not have this in the PR. A good rule of thumb is that we only PR things that are being used and are testable by the reviewer